src/main/java/ai/knowly/langtorch/llm/minimax/schema/config/MiniMaxServiceConfig.java

```diff
@@ -0,0 +1,32 @@
+package ai.knowly.langtorch.llm.minimax.schema.config;
+
+import com.google.auto.value.AutoValue;
+import java.time.Duration;
+
+/**
+ * @author maxiao
+ * @date 2023/06/07
+ */
+@AutoValue
+public abstract class MiniMaxServiceConfig {
```

Comment:
If you are adding the retry, you probably need to update this too.

example:
https://github.com/Knowly-ai/langtorch/blob/55f9b370a883e5a40bc20eaab51f1fee56fcdfb0/src/main/java/ai/knowly/langtorch/llm/openai/schema/config/OpenAIServiceConfig.java#L11

Reply:
had done